scripts/scr_weapon/scr_weapon.gml

```diff
@@ -1,85 +1,84 @@
-function scr_weapon(argument0, argument1, argument2, argument3, argument4, argument5, argument6) {
+function scr_weapon(equipment_1, equipment_2, base_group, unit_array_position, is_dreadnought, nuum, information_wanted) {
 
-	// argument0: name
-	// argument1: weapon2 name
-	// argument2: man?
-	// argument3: ID number
-	// argument4: dreaded
-	// argument5: nuum
-	// argument6: description?
+	// equipment_1: name of the first piece of equipment
+	// equipment_2: name of second piece of equipment if any
+	// base_group: the unit type defualts to true which means a marine (this needs work)
+	// unit_array_position: the position of the marine inside the tempory combat array
+	// is_dreadnought: is unit a dreadnought
+	// nuum: not a good god damn fucking clue
+	// information_wanted: what type of information do you want returned
```

Comment:
you can give bit longer names to args, for example

`equipment_1` -> `first_equipment_piece`
`base_group` -> `unit_base_group` 
etc

IMO in general, if you have to comment vars, then something is wrong